src/ResourcePacksInfoPacket.php

```diff
@@ -26,24 +27,30 @@ class ResourcePacksInfoPacket extends DataPacket implements ClientboundPacket{
 	public bool $mustAccept = false; //if true, forces client to choose between accepting packs or being disconnected
 	public bool $hasAddons = false;
 	public bool $hasScripts = false; //if true, causes disconnect for any platform that doesn't support scripts yet
+	private UuidInterface $worldTemplateId;
+	private string $worldTemplateVersion;
```

Comment:
These should be public as there are no get/setters

Reply:
You're right. If someone wants to make a PR of it, go for it. Else I'll sort it out tomorrow

Reply:
Yeah, the convention has been to keep fields public in older packets and only private them in newly added packets, to ensure the API is consistent within packets

Reply:
That being said I've never been super sure whether privating the fields was a good idea... what we really need more than anything else is #95